src/types/plugin-input.ts

```diff
@@ -39,7 +39,7 @@ function maxConcurrentTasks() {
 
 export const pluginSettingsSchema = T.Object(
   {
-    reviewDelayTolerance: T.String({ default: "1 Day" }),
+    reviewDelayTolerance: T.String({ default: "1 Day", description: "How long shall the wait be for a reviewer to take action?" }),
```

Comment:
This description isn't very clear. Given the specification, I guess that this should describe the max time we will wait for a reviewer to review while holding up the task for the assignee

Reply:
Maybe something like "Duration delay for a pull-request to count against the user's maximum assigned issues quota" ?